test/index.js

```diff
@@ -9,7 +9,7 @@ const appDefinition = require('../examples/definition.json');
 
 const copy = o => JSON.parse(JSON.stringify(o));
 
-const NUM_SCHEMAS = 47; // changes regularly as we expand
+const NUM_SCHEMAS = 48; // changes regularly as we expand
```

Comment:
😮 I didn't even know to change this! 

Total tangent for this PR: are we only counting the files in `/schemas` using `makeSchema`? Could this be pulled into a function that just looks up the dir?

Reply:
it's verifying the number of properties on the exported schema. We can't count the files because we need to verify that all the files are actually rigged up. here are the steps I went through when adding this schema:

1. write new schema
2. auto-tests pass!
3. it's not listed in docs though 🤔 
3. actually add schema to root
3. it's in the docs now, but the tests fair (need to increment this value)
3. do that, tests pass!

so I think it's good as is. We rarely add new schemas, so it hasn't been a big deal to bump as needed.

Reply:
That explanation totally makes sense—thanks @xavdid!